qemu/tests/cfg/blockdev_mirror_firewall.cfg

```diff
@@ -31,6 +31,8 @@
     image_name_data1 = data1
     image_name_mirror1 = mirror1
     rebase_mode = unsafe
+    #Fixme if no driver deprecated warning in host_dmesg.log
+    host_dmesg_ignore = yes
```

Comment:
The same concern as #3623 , it's expected to fail the tc here, when the warning msg issue is fixed, the tc passed. @aliang123  @XueqiangWei

Reply:
@zhencliu, warning msg of driver deprecated should be omitted, I think.

Reply:
what about using expected_host_dmesg, e.g. 
expected_host_dmesg = Warning: Deprecated Driver is detected
Then the related warning msgs will be ignored;

In this way, we will not ignore others if sth wrong happens

Reply:
@zhencliu, ok, this seems another solution. But once the deprecated driver issue resolved, we also need to update the cfg again.

Reply:
Also, I'm curious why we didn't hit this for other testings, this is a post check, does this testing introduce the deprecated driver only? @aliang123

Reply:
(1/1) Host_RHEL.m9.u2.ovmf.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.blockdev_mirror_firewall.q35: PASS (289.99 s)

Reply:
> Also, I'm curious why we didn't hit this for other testings, this is a post check, does this testing introduce the deprecated driver only? @aliang123
@zhencliu Yes, zhenchao, only this case can hit this issue, other cases work ok.
After checking the test result recoreds, we can see that sometimes we can hit this issue, and sometimes not.
So only define "expected_host_dmesg = Warning: Deprecated Driver is detected" here is not aviable now.
As it's just some warning msg, we can omit them.
May we need to modify them to "expected_host_dmesg = Warning: Deprecated Driver is detected | ".